### PR TITLE
Refactor data flow

### PR DESCRIPTION
**Pull request by ghost**
@philjdelong @DanielEFrampton @itemniner  Hello gentlemen I have finished refactoring the classes so the file paths are clear and clean and pretty. please take a look and give me some feedback. thank you!!

### CODE REVIEW

File: runner.rb [hunk 1]
```diff
@@ -1,14 +1 @@
...
-require_relative './lib/stat_tracker'
 
-# Using dummy data files for now, change later
-
-locations = {
-  games: './data/dummy_games.csv',
-  teams: './data/dummy_teams.csv',
-  games_teams: './data/dummy_games_teams.csv'
-}
-
```
Comment (raymondknguyen) [inline runner.rb]: please reinstate runner file

File: runner.rb [hunk 2]
```diff
@@ -1,14 +1 @@
...
-stat_tracker = StatTracker.from_csv(locations)
-
-require "pry"; binding.pry
```
Comment (DanielEFrampton) [inline runner.rb]: I think this got deleted accidentally - how about we reinstate it?

**Review by philjdelong — APPROVED** [review]

File: test/games_collection_test.rb [hunk 3]
```diff
@@ -3,41 +3,15 @@
...
   end
 
   def test_it_exists
     assert_instance_of GamesCollection, @games_collection
   end
 
   def test_it_initializes_attributes
-    assert_equal @games_array, @games_collection.games
+    assert_equal 99, @games_collection.games.length
+    assert_equal true, @games_collection.games.all? {|game| game.is_a?(Game)}
```
Comment (DanielEFrampton) [inline test/games_collection_test.rb:15]: There are a few other tests we could write to make this more robust (commenting as a reminder to myself), but this works great for now!

**Review by DanielEFrampton — APPROVED** [review]

**Review by raymondknguyen — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
I double checked

File: test/games_collection_test.rb (hunk 3 above)
Comment (ghost) [reply test/games_collection_test.rb:15]: Cool, yeah I think we should wait till we have more methods so we aren't waisting time refactoring unnecassarily

File: runner.rb (hunk 1 above)
Comment (ghost) [reply runner.rb]: Done, new runner is up to date

**Comment by ghost** [comment]
@philjdelong @itemniner @DanielEFrampton  I have refactored the requested files. please review and merge. thank you!

**Pull request merged**